### PR TITLE
move /api/user/update from model to api

### PR DESCRIPTION
**Pull request by IgorZiegler**

### CODE REVIEW

**Comment by knutole** [comment]
@IgorZiegler Looks good, OK if I merge?

**Comment by IgorZiegler** [comment]
@knutole I think yes

**Comment by IgorZiegler** [comment]
@knutole i have update it

File: js/src/models/model.users.js [hunk 1]
```diff
@@ -220,7 +220,19 @@ Wu.User = Wu.Class.extend({
 
 	_save : function (changes) {
 		if (app.activeProject) changes.project = app.activeProject.getUuid(); // for edit_user access, may need project...
-		Wu.save('/api/user/update', JSON.stringify(changes)); 
+		app.api.updateUser(JSON.stringify(changes), function (err, result) {
```
Comment (knutole) [inline js/src/models/model.users.js]: actually, here you should pass object, not stringified. got an error, and anyway it will be stringified later.

Comment (IgorZiegler) [inline js/src/models/model.users.js]: @knutole  Should I remove JSON.stringify ?

**Pull request merged**